crates/pipeline-manager/src/api/metrics.rs

```diff
@@ -0,0 +1,61 @@
+use actix_web::{
+    get,
+    http::Method,
+    web::{Data as WebData, ReqData},
+    HttpResponse,
+};
+use awc::body::MessageBody as _;
+
+use crate::{
+    api::ServerState,
+    db::{
+        storage::Storage as _,
+        types::{pipeline::PipelineStatus, tenant::TenantId},
+    },
+    error::ManagerError,
+};
+
+/// Returns the metrics of all running pipelines belonging to this tenant
```

Comment:
"Retrieve the metrics ..... to this tenant."
(For consistency across all endpoint docs)

Reply:
In addition, there should be some remark that these are fetched over time:
"The metrics are collected by making individual HTTP requests to the /metrics endpoint of each pipeline, of which only successful responses are included in the returned list."